<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>857</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string comment="Enhanced v2 Client">TheSleeper</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../thesleeper.qrc">
    <normaloff>:/images/images/biohazard.jpg</normaloff>:/images/images/biohazard.jpg</iconset>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QDockWidget" name="systemControlWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="floating">
       <bool>false</bool>
      </property>
      <property name="features">
       <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
      </property>
      <property name="allowedAreas">
       <set>Qt::AllDockWidgetAreas</set>
      </property>
      <property name="windowTitle">
       <string>Controle du système distant</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="functionalLayout">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="tabPosition">
             <enum>QTabWidget::North</enum>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::Rounded</enum>
            </property>
            <property name="currentIndex">
             <number>4</number>
            </property>
            <property name="elideMode">
             <enum>Qt::ElideNone</enum>
            </property>
            <property name="usesScrollButtons">
             <bool>false</bool>
            </property>
            <widget class="QWidget" name="systemControlTab">
             <attribute name="title">
              <string>Controle Système</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QGridLayout" name="systemControlLayout">
                <item row="0" column="3">
                 <widget class="QPushButton" name="logoutButton">
                  <property name="text">
                   <string>Fermer session</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="hibernateButton">
                  <property name="text">
                   <string>Mise en veille</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="rebootButton">
                  <property name="text">
                   <string>Redémarrer PC</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QPushButton" name="shutdownButton">
                  <property name="text">
                   <string>Eteindre PC</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="remoteShellButton">
                  <property name="text">
                   <string>Cmd.exe à distance</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="getPasswordButton">
                  <property name="text">
                   <string>Récupération mots de passes</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="taskManagerTab">
             <attribute name="title">
              <string>Gestionnaire Processus</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="taskControlLayout">
                <item>
                 <widget class="QPushButton" name="updateProcessListButton">
                  <property name="text">
                   <string>&amp;Update liste</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="killProcessButton">
                  <property name="text">
                   <string>&amp;Tuer Processus</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListView" name="processView">
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="resizeMode">
                 <enum>QListView::Adjust</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="screenshotTab">
             <attribute name="title">
              <string>Screenshot</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QPushButton" name="updateScreenButton">
                <property name="text">
                 <string>Capturer</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="saveScreenButton">
                <property name="text">
                 <string>Enregistrer</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="2">
               <widget class="QLabel" name="screenLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="2">
               <widget class="QProgressBar" name="screenProgressBar">
                <property name="value">
                 <number>0</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="textVisible">
                 <bool>true</bool>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="textDirection">
                 <enum>QProgressBar::BottomToTop</enum>
                </property>
                <property name="format">
                 <string>Téléchargement Screen : %p%</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="jpegQualitySpinBox">
                <property name="decimals">
                 <number>0</number>
                </property>
                <property name="minimum">
                 <double>1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
                <property name="value">
                 <double>50.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="qualityLabel">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Qualité du Jpeg (en %) : </string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="buddy">
                 <cstring>jpegQualitySpinBox</cstring>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="keyloggerTab">
             <attribute name="title">
              <string>Keylogger</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="keyloggerLayout">
                <item>
                 <widget class="QPushButton" name="updateLogButton">
                  <property name="text">
                   <string>&amp;Recuperer fichier Log</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="saveLogButton">
                  <property name="text">
                   <string>Enregistrer</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="deleteLogButton">
                  <property name="text">
                   <string>Effacer</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTextEdit" name="keylogEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="undoRedoEnabled">
                 <bool>false</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::TextSelectableByMouse</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QProgressBar" name="keylogProgressBar">
                <property name="value">
                 <number>0</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="invertedAppearance">
                 <bool>false</bool>
                </property>
                <property name="format">
                 <string>Téléchargement Keylog : %p%</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="ftpTab">
             <attribute name="title">
              <string>Transferts de fichiers</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QGroupBox" name="serverFilesBox">
                  <property name="title">
                   <string>Fichiers distants</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_3">
                   <item row="0" column="0">
                    <widget class="QTreeView" name="remoteFilesView">
                     <property name="editTriggers">
                      <set>QAbstractItemView::NoEditTriggers</set>
                     </property>
                     <property name="rootIsDecorated">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="clientFilesBox">
                  <property name="title">
                   <string>Fichiers locaux</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QTreeView" name="localFilesView">
                     <property name="editTriggers">
                      <set>QAbstractItemView::NoEditTriggers</set>
                     </property>
                     <property name="rootIsDecorated">
                      <bool>false</bool>
                     </property>
                     <property name="sortingEnabled">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="downloadButton">
                  <property name="text">
                   <string>&amp;Download</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="uploadButton">
                  <property name="text">
                   <string>&amp;Upload</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QProgressBar" name="transferProgressBar">
                <property name="value">
                 <number>0</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClient">
    <property name="title">
     <string>&amp;Client</string>
    </property>
    <addaction name="actionShutdownServer"/>
    <addaction name="actionRebootServer"/>
    <addaction name="actionUninstallServer"/>
    <addaction name="actionUpgradeServer"/>
   </widget>
   <widget class="QMenu" name="menuServeur">
    <property name="title">
     <string>&amp;Serveur</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>&amp;Aide</string>
    </property>
    <addaction name="actionTutorial_TheSleeper"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAboutInnocence"/>
   </widget>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionEditerClient"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuClient"/>
   <addaction name="menuServeur"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QDockWidget" name="connectionsManagerWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Gestions des serveurs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="serversTreeLayout">
       <item>
        <widget class="QTableView" name="connectedClientsView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideMiddle</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="disconnectButton">
           <property name="text">
            <string>&amp;Deconnecter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="listenButton">
           <property name="text">
            <string>&amp;Ecouter Connexions</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="netconfigLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="portLabel">
           <property name="text">
            <string>&amp;Port : </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buddy">
            <cstring>portBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="portBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>32768</number>
           </property>
           <property name="value">
            <number>80</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="mdpLabel">
           <property name="text">
            <string>&amp;Mot de passe : </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buddy">
            <cstring>passEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="passEdit">
           <property name="text">
            <string>crunch</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionConsoleDebug">
   <property name="text">
    <string>Console Debug</string>
   </property>
  </action>
  <action name="actionShutdownServer">
   <property name="text">
    <string>Tuer </string>
   </property>
  </action>
  <action name="actionRebootServer">
   <property name="text">
    <string>Redémarrer</string>
   </property>
  </action>
  <action name="actionUninstallServer">
   <property name="text">
    <string>Désinstaller</string>
   </property>
  </action>
  <action name="actionUpgradeServer">
   <property name="text">
    <string>Mettre à jour</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configurer</string>
   </property>
  </action>
  <action name="actionAboutInnocence">
   <property name="text">
    <string>About Innocence</string>
   </property>
  </action>
  <action name="actionEditer_client">
   <property name="text">
    <string>Editer Client</string>
   </property>
  </action>
  <action name="actionNouveau_Client">
   <property name="text">
    <string>Nouveau Client</string>
   </property>
  </action>
  <action name="actionEditerClient">
   <property name="text">
    <string>Editer Client</string>
   </property>
  </action>
  <action name="actionTutorial_TheSleeper">
   <property name="text">
    <string>Tutorial TheSleeper</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>shutdownButton</tabstop>
  <tabstop>rebootButton</tabstop>
  <tabstop>hibernateButton</tabstop>
  <tabstop>logoutButton</tabstop>
  <tabstop>remoteShellButton</tabstop>
  <tabstop>getPasswordButton</tabstop>
  <tabstop>updateProcessListButton</tabstop>
  <tabstop>killProcessButton</tabstop>
  <tabstop>updateScreenButton</tabstop>
  <tabstop>saveScreenButton</tabstop>
  <tabstop>jpegQualitySpinBox</tabstop>
  <tabstop>updateLogButton</tabstop>
  <tabstop>saveLogButton</tabstop>
  <tabstop>deleteLogButton</tabstop>
  <tabstop>downloadButton</tabstop>
  <tabstop>uploadButton</tabstop>
  <tabstop>disconnectButton</tabstop>
  <tabstop>listenButton</tabstop>
  <tabstop>portBox</tabstop>
  <tabstop>passEdit</tabstop>
  <tabstop>connectedClientsView</tabstop>
  <tabstop>remoteFilesView</tabstop>
  <tabstop>localFilesView</tabstop>
  <tabstop>keylogEdit</tabstop>
  <tabstop>processView</tabstop>
 </tabstops>
 <resources>
  <include location="../thesleeper.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteLogButton</sender>
   <signal>clicked()</signal>
   <receiver>keylogEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
